core/domain/collection_domain.py

```diff
@@ -796,7 +806,7 @@ def _convert_collection_contents_v3_dict_to_v4_dict(
             for skill_name, skill_id in skill_names_to_ids.iteritems()
         }
 
-        collection_contents['next_skill_id'] = len(skill_names)
+        collection_contents['next_skill_index'] = len(skill_names)
```

Comment:
Ditto. Don't change anything related to migrations. This will mess things up.

Reply:
Updated.